lbz/jwt_utils.py

```diff
@@ -47,8 +47,8 @@ def decode_jwt(auth_jwt_token: str) -> dict:
             raise Unauthorized(f"Your token has expired. Please refresh it.")
         except JWTError:
             raise Unauthorized
-        except Exception:
-            logger.error(f"Error during decoding, token={auth_jwt_token}")
-            raise ServerError
+        except Exception as ex:
+            msg = f"Error during decoding, token={auth_jwt_token}"
+            raise RuntimeError(msg) from ex
```

Comment:
FYI: It will be handled a level higher as `ServerError` but the log is not repeated anymore.

Reply:
I'd adjust the message in passing :wink: 

`An error occurred during decoding the token. \nToken body: \n{auth_jwt_token}` or equivalent